Lighthouse Camera/UI/Main/TTSHelper.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TTSHelper.swift
```

Comment:
This was part of refactor to bring out code from view controller in separate files, but you are right it can be a separate patch